mkosi.images/system/mkosi.conf.d/10-fedora.conf

```diff
@@ -8,4 +8,5 @@ Packages=
         btrfs-progs
         compsize
         f2fs-tools
+        glibc-all-langpacks
```

Comment:
Shouldn't installing just `glibc-langpack-en` be enough? Installing all language packs seems unnecessary, especially since that package is over 200M.

Reply:
I would have assumed so, but after substituting that `localectl list-locales` only lists

```
C.UTF-8
en_CA.UTF-8
en_DK.UTF-8
```

the files in /usr/lib/locale seem to be installed so I'm investigating why this is failing.

Reply:
```strace
openat(AT_FDCWD, "/usr/lib/locale/en_US.utf8/LC_MESSAGES/SYS_LC_MESSAGES", O_RDONLY|O_CLOEXEC) = -1 EUCLEAN (Structure needs cleaning)
```
```dmesg
[ 1628.288323] erofs: (device dm-0): erofs_read_inode: bogus i_mode (0) @ nid 0
```

Seems to have been erofs corruption.

Reply:
This doesn't appear to be a random corruption, as it consistently happens when building images using glibc-langpack-en.
The locale archive installed by glibc-all-langpacks doesn't have this problem.

Reply:
Which version of erofs-utils are you using? could you check the image directly with fsck.erofs -d9?

Reply:
> I can't reproduce the erofs error if I add `glibc-langpack-en`. (mkfs.erofs 1.7.1)

Only mkfs.erofs 1.7 had a known similar issue but it's already fixed in 1.7.1.

Reply:
Looks like it was mkfs.erofs 1.7, I'll try upgrading, thanks.

Reply:
That's fixed glibc-langpack-en.